data/help.txt

```diff
@@ -80,6 +80,16 @@ help "outfitter"
 	`Use your scroll wheel, or click and drag, to scroll the view.`
 	`As in the trading panel, you can hold down Shift or Control to buy 5 or 20 of an outfit at once, or both keys to buy 100.`
 
+help "outfitter 2"
+	`Sometimes you may want to buy some outfits directly into your cargo hold instead of having them installed. Perhaps a mission has asked you to retrieve some outfits or you want to carry extra ammunition to refill your ships. There are several ways you can do so:`
+	`1) The outfitter interface has a small checkbox labeled 'in-cargo' that, when checked, will instruct the dockworkers to load any outfits you purchase into your cargo bay instead of installing it.`
+	`2) If you do not have any ships selected in the outfitter control panel when you make a purchase, the outfitter will assign it for delivery to your cargo hold instead of scheduling an installation.`
```

Comment:
Instead of basically copy-pasting from the Remnant outfitter, I'd say reword these so that they fit the style of the other help dialogs. That is, they aren't written "in-universe" but instead from the perspective of someone playing a game.

Reply:
@Amazinite OK, I changed a  bunch of these two texts. I didn't really think that either one of them was overly in-character, just something that wasn't explicitly fourth-wall breaking. But either way, they're somewhat different now.